ui/lib/core/addon/utils/client-count-utils.js

```diff
@@ -0,0 +1,113 @@
+import { parseAPITimestamp } from 'core/utils/date-formatters';
+
+export const flattenDataset = (object) => {
+  if (Object.values(object).every((e) => e === null)) return object;
+  let flattenedObject = {};
+  Object.keys(object['counts']).forEach((key) => (flattenedObject[key] = object['counts'][key]));
+  return homogenizeClientNaming(flattenedObject);
+};
+
+export const formatByMonths = (monthsArray) => {
```

Comment:
TODO: this function will be refactored now that we have a way to generate mock data with the real API